### PR TITLE
Fix some orchestration flags

### PR DESCRIPTION
**Pull request by inercia**
* Do not set the `bootstrap_complete` flag in all the nodes: do it only in the nodes that had some role assigned.
  This is important because, when some nodes are _unassigned_ and we bootstrap the cluster, those nodes will have the `bootstrap_complete` set anyway.
  The problem appears when running the _node removal_ orchestration, as it uses that flag for prioritizing nodes when looking for a replacement (it tries to skip nodes with `bootstrap_complete` as it assumes that some role is already assigned and running in those machines).
* Remove the `bootstrap_in_progress` despite the result of the orchestration (as [we currently do in the node removal orchestration](https://github.com/kubic-project/salt/blob/master/salt/orch/removal.sls#L342-L351)... and we should do in the [upddate orchestration too](https://github.com/kubic-project/salt/blob/master/salt/orch/update.sls#L441-L451)).
* Fixed typo in target for disabling the `rebootmngr`

### CODE REVIEW

File: salt/orch/kubernetes.sls [hunk 1]
```diff
@@ -12,10 +12,11 @@
 {%- set additional_etcd_members = salt.caasp_etcd.get_additional_etcd_members(num_wanted=num_etcd_members,
                                                                               etcd_members=etcd_members) %}
 
-# Ensure the node is marked as bootstrapping
+# Ensure all the nodes are marked with a 'bootstrap_in_progress' flag
 set-bootstrap-in-progress-flag:
   salt.function:
-    - tgt: '*'
+    - tgt: 'roles:(ca|admin|kube-master|minion|etcd)'
```
Comment (ereslibre) [inline salt/orch/kubernetes.sls]: Typo here: `|minion|`

Comment (inercia) [reply salt/orch/kubernetes.sls]: I did the same typo I fixed some lines below... (faceplam)

Fixing.

File: salt/orch/kubernetes.sls [hunk 2]
```diff
@@ -12,10 +12,11 @@
 {%- set additional_etcd_members = salt.caasp_etcd.get_additional_etcd_members(num_wanted=num_etcd_members,
                                                                               etcd_members=etcd_members) %}
 
-# Ensure the node is marked as bootstrapping
+# Ensure all the nodes are marked with a 'bootstrap_in_progress' flag
 set-bootstrap-in-progress-flag:
   salt.function:
-    - tgt: '*'
+    - tgt: 'roles:(ca|admin|kube-master|kube-minion|etcd)'
```
Comment (ereslibre) [inline salt/orch/kubernetes.sls]: I'm not sure if we should include the `ca` in this set... It won't harm though I guess.

Comment (inercia) [reply salt/orch/kubernetes.sls]: Yeah, I thought the same...

**Pull request merged**